src/interpreter/Interpreter.ts

```diff
@@ -226,6 +228,38 @@ export class Interpreter {
       case AstNodeType.PARENTHESIS: {
         return this.evaluateAst(ast.expression, state)
       }
+      case AstNodeType.MATRIX: {
+        let totalWidth: Maybe<number> = undefined
+        const ret: InternalScalarValue[][] = []
+        for(const astRow of ast.args) {
+          let rowHeight: Maybe<number> = undefined
+          const rowRet: InternalScalarValue[][] = []
+          for(const astIt of astRow) {
+            const arr = coerceToRange(this.evaluateAst(astIt, state))
+            const height = arr.height()
+            if(rowHeight===undefined) {
+              rowHeight = height
+              rowRet.push(...arr.data)
+            } else if(rowHeight === height) {
+              for(let i=0;i<height;i++) {
+                rowRet[i].push(...arr.data[i])
+              }
+            } else {
+              return new CellError(ErrorType.REF, ErrorMessage.SizeMismatch)
```

Comment:
I would say it's `ErrorType.VALUE`

Reply:
<img width="404" alt="Screenshot 2021-05-19 at 10 44 31" src="https://user-images.githubusercontent.com/40573492/118783272-342b0200-b88f-11eb-81b1-948b28214f2a.png">